packages/node/src/lib/events/events.ts

```diff
@@ -17,6 +17,7 @@ export class Events extends WithHttp implements IEvents {
       transactionId: data.transactionId,
       overrides: data.overrides || {},
       ...(data.actor && { actor: data.actor }),
+      data,
```

Comment:
@gitstart this is not the right place for it. The data of the subscriber will be already included inside the data.to. 
The change you need to make will be inside the process-subscriber usecase. I believe it is not passed to the create subscriber there.